packages/ckeditor5-clipboard/tests/dragdropexperimental.js

```diff
@@ -0,0 +1,2241 @@
+					'<paragraph>[]foobar</paragraph>' +
+					'<horizontalLine></horizontalLine>'
+				);
+
+				const clock = sinon.useFakeTimers();
+				const widgetViewElement = viewDocument.getRoot().getChild( 1 );
+				const viewElement = widgetViewElement.getChild( 0 );
+				const domNode = domConverter.mapViewToDom( viewElement );
+
+				expect( viewElement.is( 'element', 'hr' ) ).to.be.true;
+
+				const eventData = {
+					domTarget: domNode,
+					target: viewElement,
+					domEvent: {}
+				};
+
+				viewDocument.fire( 'mousedown', {
+					...eventData
+				} );
+
+				expect( widgetViewElement.getAttribute( 'draggable' ) ).to.equal( 'true' );
+
+				viewDocument.fire( 'mouseup' );
+				clock.tick( 50 );
+
+				expect( widgetViewElement.hasAttribute( 'draggable' ) ).to.be.false;
+			} );
+
+			it( 'can drag multiple elements', () => {
+				setModelData( model,
+					'<blockQuote>' +
+						'[<paragraph>foo</paragraph>' +
+						'<paragraph>bar</paragraph>]' +
+						'<paragraph>baz</paragraph>' +
+					'</blockQuote>' +
+					'<horizontalLine></horizontalLine>'
+				);
+
+				const dataTransferMock = createDataTransfer();
+				const positionAfterHr = model.createPositionAt( root.getChild( 1 ), 'after' );
+
+				fireDragStart( dataTransferMock );
+				expectDragStarted( dataTransferMock, '<p>foo</p><p>bar</p>' );
+
+				fireDragging( dataTransferMock, positionAfterHr );
+				expectDraggingMarker( positionAfterHr );
+			} );
+
+			it( 'should remove "draggable" attribute from editable element', () => {
+				setModelData( model, '<paragraph>[foo]bar</paragraph>' );
+
+				const clock = sinon.useFakeTimers();
+				const editableElement = viewDocument.getRoot();
+				const viewElement = editableElement.getChild( 0 );
+				const domNode = domConverter.mapViewToDom( viewElement );
+
+				expect( viewElement.is( 'element', 'p' ) ).to.be.true;
+
+				const eventData = {
+					domTarget: domNode,
+					target: viewElement,
+					domEvent: {}
+				};
+
+				viewDocument.fire( 'mousedown', {
+					...eventData
+				} );
+
+				expect( editableElement.getAttribute( 'draggable' ) ).to.equal( 'true' );
+
+				viewDocument.fire( 'mouseup' );
+				clock.tick( 50 );
+
+				expect( editableElement.hasAttribute( 'draggable' ) ).to.be.false;
+			} );
+
+			it( 'should only show one preview element', () => {
+				setModelData( model,
+					'<blockQuote>' +
+						'[<paragraph>foo</paragraph>' +
+						'<paragraph>bar</paragraph>]' +
+						'<paragraph>baz</paragraph>' +
+					'</blockQuote>' +
+					'<horizontalLine></horizontalLine>'
+				);
+
+				const dataTransferMock = createDataTransfer();
+
+				fireDragStart( dataTransferMock );
+				fireDragStart( dataTransferMock );
+
+				const numberOfCkContentElements = Object
+					.keys( document.getElementsByClassName( 'ck-content' ) )
+					.length;
+
+				// There should be two elements with the `.ck-content` class - editor and drag-and-drop preview.
+				expect( numberOfCkContentElements ).to.be.eq( 2 );
```

Comment:
```suggestion
				expect( numberOfCkContentElements ).to.equal( 2 );
```
Is there any difference? If not, let's be consistent in style.